### DISCUSSION TITLE
Folders can't be uploaded in repository

### DISCUSSION THREAD

**Discussion by feintSlash** [community/community#52017] [Repositories]
### Select Topic Area

General

### Body

**Comment by airtower-luna** [comment]
I'm afraid you'll have to be a lot more specific for anyone to be able to help. A good problem description should include at least:

* What exactly are you doing?
* What happens?
* What would you expect to happen instead?

Try to be as precise as possible.

**Reply by airtower-luna** [reply]
I don't mean what you project is, but exact steps your taking. Which command(s) or UI elements do you use when trying to upload your folders? What *exact and full* error messages do you get? Those are the kinds of details people need to be able to help you.

**Comment by YasinDehfuli** [comment]
You should uninstall node_modules or any other folders that have large size
